packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -58,7 +58,6 @@
 
 	.dataviews-view-grid__media {
 		width: 100%;
-		min-height: 200px;
```

Comment:
Possibly a naive question: do we not need a `min-height` because of the `aspect-ratio` being set? Since we're adding in a `120` value I was wondering at first if a `min-height` of `120px` would make sense, but so far in testing I haven't run into any issues with this removal 👍

Reply:
Yes, we don't need min-height due to aspect-ratio. I'm not sure why it was added initially, but removing it doesn't seem to break anything.